docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -1642,6 +1642,18 @@ Result:
 **See Also**
 - [date_trunc](#date_trunc)
 
+## toStartOfIntervalAllowNegative
```

Comment:
Let’s also add a note to the `toStartOfInterval` function documentation, indicating that there's `toStartOfIntervalAllowNegative` that should be used in such cases